### PR TITLE
chore: add openapi open-payments http-signature-utils libraries

### PR DESCRIPTION
**Pull request by mkurapov**
## Changes proposed in this pull request

- Copying over `open-payments` `openapi` `http-signature-utils` into this repo & adding corresponding CI steps
- Adding auxiliary repo files (pnpm config, prettier, eslint, etc), all copied over from rafiki

Notes:
- Next steps will be adding lerna to this repo to prep packages for publish

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,17 +15,17 @@ including:
 - Subscriptions
```
Comment (mkurapov) [inline README.md:15]: only prettier formatting changes

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.next
+public
+generated
+dist
+build
+landing
```
Comment (mkurapov) [inline .eslintignore:6]: Going to ignore `landing` for now, will be a separate PR

File: landing/package.json [hunk 3]
```diff
@@ -1,5 +1,4 @@
 {
-  "name": "open-payments",
```
Comment (mkurapov) [inline landing/package.json:2]: don't need package name, since it conflicts with `open-payments` package

File: pnpm-workspace.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+packages:
+  - 'packages/*'
+  - 'landing'
```
Comment (mkurapov) [inline pnpm-workspace.yaml:3]: Do we want to move `landing` under `packages/landing` ?

File: packages/openapi/README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+# OpenAPI 3.1 Validator
```
Comment (wilsonianb) [inline packages/openapi/README.md:1]: `openapi` _could_ be in a standalone repo since it's not Open Payments specific despite using and Open Payments spec for testing

File: package.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    "check:lint": "eslint --max-warnings=0 .",
+    "check:prettier": "prettier --check .",
+    "clean": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
+    "build": "tsc --build"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^17.4.4",
+    "@commitlint/config-conventional": "^17.4.4",
+    "@jest/types": "^29.5.0",
+    "@swc/core": "1.3.39",
```
[the remark this answers is not in the document]
Comment (wilsonianb) [reply package.json:24]: Are there any `*.tsx` files?

[the remark this answers is not in the document]
Comment (mkurapov) [reply package.json:24]: Could probably remove it from rafiki too

[the remark this answers is not in the document]
Comment (mkurapov) [reply package.json:24]: Interestingly enough that does seem to be the proper config: https://swc.rs/docs/usage/jest#installation

File: packages/openapi/README.md (hunk 5 above)
Comment (mkurapov) [reply packages/openapi/README.md:1]: I think it's fine in here for now to keep things simple

**Review by sabineschaller — APPROVED** [review]
Let's merge this and add potential fixes in a PR.

File: pnpm-workspace.yaml (hunk 4 above)
Comment (mkurapov) [reply pnpm-workspace.yaml:3]: Sounds good, it's https://openpayments.guide/ site, so still relevant?
